examples/step-60/doc/intro.dox

```diff
@@ -60,19 +60,20 @@ embedded domain (`dim`) is always smaller by one or equal with respect to the
 dimension of the embedding domain $\Omega$ (`spacedim`).
 
 We are going to solve the following differential problem: given a sufficiently
-regular function $g$ on $\Gamma$, find the solution $u$ to
+regular function $g$ on $\Gamma$, a forcing term $f \in L^2(\Omega)$ and a Dirichlet data
```

Comment:
> a Dirichlet data

This doesn't read right to me. Do you mean "Dirichlet conditions", or something else?

Reply:
Yes, exactly. I could replace that with "a Dirichlet boundary condition", since that is imposed on dOmega only

Reply:
Yes, that would be perfect thanks!